### ISSUE TITLE
hyperparameters for mask rcnn in .config

### ISSUE THREAD

**Issue by angyee** [vijendra1125/Custom-Mask-RCNN-using-Tensorfow-Object-detection-API#27]
my objective is to detect the boxes and totes, but the pre-trained model is trained on coco data set, to detect my objects, whats need to be changed as batch size and learning rate and? what it is value?
![brown_boxes_stack_tile_1706294410](https://user-images.githubusercontent.com/17041720/76406545-e758bb00-63af-11ea-964d-b05905921c08.png)
![totes](https://user-images.githubusercontent.com/17041720/76406549-e889e800-63af-11ea-8e50-fc43457f04ff.jpeg)
